tests/Makefile.conf

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
We probably don't need to check this one in?

Reply:
Ah! Yes, thanks for catching that.

I've pushed an update :)